presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/ValuesMatcher.java

```diff
@@ -91,12 +94,12 @@ public MatchResult detailMatches(PlanNode node, StatsProvider stats, Session ses
                                 return new DoubleLiteral(String.valueOf(expression.getValue()));
                             }
                             if (expression.getType().getJavaType() == Slice.class) {
-                                return new StringLiteral(String.valueOf(expression.getValue()));
+                                return new StringLiteral(((Slice) expression.getValue()).toStringAscii());
```

Comment:
actually, why not `toStringUtf8`?

Reply:
Good point. `toStringAscii` comes first in Intellij auto-complete 🙃